docs/socialaccount/providers/saml.rst

```diff
@@ -72,21 +72,37 @@ via the Django admin as well:
                         # If left empty, an attempt will be done to map the attributes using
                         # built-in defaults.
                         "attribute_mapping": {
-                            "uid": "http://schemas.auth0.com/clientID",
-                            "email_verified": "http://schemas.auth0.com/email_verified",
```

Comment:
**Issue**: I think we need to keep this one...

Reply:
Ok, I committed your suggestions.

I just thought it was easy to overlook that they are alternative methods (happened to me), so I wanted to highlight it.